api/src/models/navLinks.schema.ts

```diff
@@ -0,0 +1,6 @@
+import * as mongoose from 'mongoose';
```

Comment:
name this file consistently, the resource called `links`, then lal files, modules or related code should be links not `navLinks`